tsconfig.json

```diff
@@ -1,6 +1,5 @@
 {
   "compilerOptions": {
-    "rootDir": ".",
```

Comment:
was this breaking anything? 🤔

Reply:
that's weird as the tsconfig is at the root of all packages. Maybe we should add another local tsconfig that extends this one